k8s/aat/common/ccd/data-store-api.yaml

```diff
@@ -14,14 +14,12 @@ spec:
   chart:
     repository: https://hmctspublic.azurecr.io/helm/v1/repo/
     name: ccd-data-store-api
```

Comment:
@hemantt this can only be merged after we release on Prod https://github.com/hmcts/ccd-data-store-api/pull/773/files